### PR TITLE
VAM-16: XUS GET USER INFO RPC/Service Implementation

### PR DESCRIPTION
**Pull request by mfuroyama**
This branch includes the following modules and tests:
### Services
* userMetaService
    * getUserInfo
* systemMetaService
    * getKernelSystemParameters

### Models
* XUS GET USER INFO

### CODE REVIEW

File: lib/broker/models/xus-get-user-info.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+    invokeRPC: (rpcArgs, session, callback) => {
+        if (!session || !session.userId || !session.stationId) {
+            return callback(new Error('Invalid session context'));
+        }
+        userService.getUserInfo(session, callback);
+    },
+    formatResult: (res) => {
+        if (!_.isObject(res)) {
+            return new Error('Invalid response: Respons must be an object');
```
Comment (creindle) [inline lib/broker/models/xus-get-user-info.js]: `return new Error('Invalid response: **Response** must be an object');` Sorry! Just caught my eye ;D

Comment (creindle) [reply lib/broker/models/xus-get-user-info.js]: Yay it did show up! Also, sorry, I don't know if I should've just made a commit... but that's not how these work correct? We don't write commits on other pull requests correct?

Comment (kekuso) [reply lib/broker/models/xus-get-user-info.js]: Christie = designated spell checker :D

Comment (mfuroyama) [reply lib/broker/models/xus-get-user-info.js]: Thanks for catching this!

**Pull request merged**